src/lib/eslint/CustomEslintEngine.ts

```diff
@@ -82,22 +82,26 @@ export class CustomEslintEngine extends AbstractRuleEngine {
 			this.eventCreator.createUxInfoAlwaysMessage('info.filtersIgnoredCustom', []);
 		}
 
-		const cli = this.dependencies.createCLIEngine(config);
+		const eslint = this.dependencies.createESLint(config);
 
 		const results: RuleResult[] = [];
 		for (const target of targets) {
-			const report: ESReport = cli.executeOnFiles(target.paths);
+			const esResults: ESResult[] = await eslint.lintFiles(target.paths);
```

Comment:
`lintFiles()` returns an array of results directly, instead of wrapping them in an `ESReport`.